### PR TITLE
Add RPC WS options to specify password file for keystore and truststore

### PR DESCRIPTION
**Pull request by pullurib**
### PR Description
- Introduces options to provide keystore and truststore password files.
- Add new options
- Validate options 
- Modify WebSocket configuration 
- Update test cases 

## Fixed Issue(s)

#7931

### CODE REVIEW

**Review by fab-10 — APPROVED** [review]
Looks good, just update the CHANGELOG

File: besu/src/test/resources/everything_config.toml [hunk 1]
```diff
@@ -123,10 +123,12 @@ rpc-ws-authentication-jwt-public-key-file="none"
 rpc-ws-ssl-enabled=false
 rpc-ws-ssl-keystore-file="none.pfx"
 rpc-ws-ssl-keystore-password="none.passwd"
+rpc-ws-ssl-keystore-password-file="none.txt"
 rpc-ws-ssl-keystore-type="none"
 rpc-ws-ssl-client-auth-enabled=false
 rpc-ws-ssl-truststore-file="none.pfx"
 rpc-ws-ssl-truststore-password="none.passwd"
+rpc-ws-ssl-truststore-password-file="none.txt"
```
Comment (macfarla) [inline besu/src/test/resources/everything_config.toml:131]: does it make sense to specify both password and password-file or should we only allow one or the other?

Comment (fab-10) [reply besu/src/test/resources/everything_config.toml:131]: that should be handled by the ArgGroup

**Comment by fab-10** [comment]
@pullurib please check if this failing test is related to your change
```
BesuCommandWithRequiredOptionsTest > presentRequiredOptionShouldPass() FAILED
    java.lang.AssertionError at BesuCommandWithRequiredOptionsTest.java:31
```

**Comment by pullurib** [comment]
They pass when I sync my local branch and run the tests locally. I do see an error message though 

2024-12-04 10:15:30.291-05:00 | Test worker | INFO  | VersionMetadata | Existing version data detected. Besu version 00.0-develop-xxxxxxxx
2024-12-04 10:15:30.777-05:00 | Test worker | INFO  | KZGPointEvalPrecompiledContract | Loading network trusted setup from classpath resource /kzg-trusted-setups/mainnet.txt
2024-12-04 10:15:30.894-05:00 | Test worker | INFO  | CommandTestAbstract | Standard output 
2024-12-04 10:15:30.895-05:00 | Test worker | INFO  | CommandTestAbstract | Standard error 
2024-12-04 10:15:31.060-05:00 | Test worker | INFO  | CommandTestAbstract | Standard output 
2024-12-04 10:15:31.060-05:00 | Test worker | INFO  | CommandTestAbstract | Standard error Missing required option: '--accept-terms-and-conditions=<acceptTermsAndConditions>'

To display full help:
besu [COMMAND] --help

2024-12-04 10:15:31.098-05:00 | Test worker | INFO  | CommandTestAbstract | Standard output 
2024-12-04 10:15:31.098-05:00 | Test worker | INFO  | CommandTestAbstract | Standard error Missing required option: '--accept-terms-and-conditions=<acceptTermsAndConditions>'

To display full help:
besu [COMMAND] --help

**Pull request merged**